common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -133,6 +133,7 @@ public ScoreResponse score(ScoreRequest request) {
     } catch (Exception e) {
       log.info("Failed shapley contribution due to: {}", e.getMessage());
       log.debug(" - failure cause: ", e);
```

Comment:
side note:  generally, suggestion is to either log or throw, but not both.  eventually, whatever part of the code catches all bubbled up exceptions would do the logging.  And with all nested exceptions simply tacking on their own messages (e.g., `throw new Illegal....Exception("Failed to ...", e)`), it all gets wrapped up into a single object that can be logged by one catcher.

Else, error/warning/etc. messages are scattered across logs (often separated by regular, healthy, logs) instead of in a single, trackable, block.

Reply:
sure that can be moved, I tried to minimize my impact to the logs in general as much as possible. The goal was to actually give the response something useful that can be used as opposed to always sending `400` and nothing else.